hack/e2e-kind-cluster-setup.sh

```diff
@@ -25,18 +30,47 @@ setup_cluster() {
 push_local_image() {
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make manifests
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make img-build
-    kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+
+    if [[ $CLUSTER_TYPE == "kind" ]]; then
+      kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+    else
+      "$OCI_BIN" push $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG 
+    fi
 }
 
 cleanup() {
     rm -rf multus-cni
     git checkout -- manifests/
 }
 
+install_multus() {
+  if [[ $MULTUS_VERSION == "latest" ]]; then
+    echo "error: MULTUS_VERSION cant be latest, when using non default CLUSTER_TYPE $CLUSTER_TYPE"
```

Comment:
```suggestion
    echo "error: kind cluster is already shipping latest multus (built from src). No point installing it
```

Reply:
it is not accurate, we reach here only if `CLUSTER_TYPE` isn't kind